.editorconfig

```diff
@@ -7,3 +7,6 @@ root = true
 indent_style = space
 indent_size = 2
 
+# Based on Google's C# style guide
```

Comment:
4 space indents will make this comment no longer true!

Reply:
Good point, this is from an earlier version where I tried to mimic the original style